api/autoscaling/v2/horizontalpodautoscaler_webhook.go

```diff
@@ -64,6 +64,9 @@ var _ admission.CustomDefaulter = &HPAWebhook{}
 func (h *HPAWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	hpa := obj.(*v2.HorizontalPodAutoscaler)
 	tortoiseName, ok := hpa.GetAnnotations()[annotation.TortoiseNameAnnotation]
+	if !ok {
+		tortoiseName, ok = hpa.GetAnnotations()[annotation.DeprecatedTortoiseNameAnnotation]
```

Comment:
To ignore a linter.
```suggestion
		//nolint // We use the deprecated annotation deliberately so that we don't break existing users.
		tortoiseName, ok = hpa.GetAnnotations()[annotation.DeprecatedTortoiseNameAnnotation]
```